### DISCUSSION TITLE
Is it possible to setup download-only/upload-only/bidirectional sync for specific folders?

### DISCUSSION THREAD

**Discussion by divideby** [abraunegg/onedrive#1835] [Q&A] [answered]
I want to upload-only from some folders, and bidirectional sync for others. Is it possible?

**Comment by abraunegg** [comment, marked as answer]
@divideby 
> I want to upload-only from some folders, and bidirectional sync for others. Is it possible?

Yes - use separate client configuration directories for each sync operation - read: https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md#configuring-the-client-to-use-multiple-onedrive-accounts--configurations
